components/typography/stories/template.js

```diff
@@ -21,6 +21,7 @@ export const Template = (args = {}, context = {}) => {
 		customClasses = [],
 		customStyles = {},
 		skipLineBreak = false,
+		lang = context.globals?.lang || "en-US",
```

Comment:
@castastrophe What do you think of this approach? The `lang` can still be passed in as an arg so that I can set multiple different `lang`s within a story, but if it's not passed in we'll try to find it in the global context or default to English.

Reply:
I love it!  Maximum flexibility.